bower.json

```diff
@@ -25,5 +25,8 @@
     "ember-cli-shims": "stefanpenner/ember-cli-shims#0.0.3",
     "jquery": "~2.1.1",
     "ember-data": "~1.0.0-beta.10"
+  },
+  "resolutions": {
+    "ember-data": "canary"
```

Comment:
I wonder if @rwjblue had meant to include `"ember-data": "release"` here instead of canary.  The current travis failures seem canary related...